os/getting-started-with-docker.md

```diff
@@ -127,4 +127,4 @@ docker run -d -p 80:80 registry.example.com:5000/apache /usr/sbin/apache2ctl -D
 #### More Information
 <a class="btn btn-default" href="{{site.baseurl}}/using-coreos/docker">Docker Overview</a>
 <a class="btn btn-default" href="http://www.docker.com/">Docker Website</a>
-<a class="btn btn-default" href="http://www.docker.com/gettingstarted/">docker's Getting Started Guide</a>
+<a class="btn btn-default" href="https://docs.docker.com/mac/started/">docker's Getting Started Guide</a>
```

Comment:
I don't really want to link to a Mac-specific link, but I couldn't find a better one.